src/main/kotlin/baseball/IOHandler.kt

```diff
@@ -0,0 +1,41 @@
+package baseball
+
+import baseball.data.GameNumDeck
+import baseball.data.InputType
+import camp.nextstep.edu.missionutils.Console
+
+class IOHandler {
+
+    private val validInputAfterGame = listOf("1", "2")
+    private val validInputWhileGame = (1..10).map { (it + ASCII_0_CODE).toChar() }
```

Comment:
```suggestion
   private val validInputWhileGame = (1..10).map { '0' + it }
```
`숫자 + 문자`의 형식에서 `문자 + 숫자`의 형식으로 순서를 바꾸면 .toChar()를 제거할 수 있을 것 같습니다.
'0'은 물론 기존처럼 변수로 받아오면 될 것 같고요!

Reply:
9-10 라인의 매직 넘버와 리터럴을 변수로 관리하는 것에 대해 어떻게 생각하시나요?

Reply:
확인이 늦어서 죄송합니다. 

### toChar 없애기
Kotlin에서 Char에 int를 더하면 자동으로 변환해주는군요!
Java로 컴파일 된 코드를 확인해보니 동일한 코드로 변경되네요. Intellij에서 View - Show ByteCodes를 실행하면 Java로 변환된 코드를 볼 수 있어요.

![code](https://github.com/woowacourse-precourse/kotlin-baseball-6/assets/48354989/3b473f9f-5539-445e-a346-a9e169aa4b43)

### 매직 넘버, 리터럴

9-10 라인에 있는 숫자들도 상수로 선언하는 것이 좋을 것 같아요!

Reply:
10번 라인에 1 ~ 10이 아니라 1 ~ 9가 올바른 범위 같네요😅

Reply:
네네 '+' 연산시에 앞에 변수의 자료형에 맞게 뒤에 변수 자료형을 바꾸는 걸로 알고 있었어요!

저는 Androud Studio를 쓰는데 여기에도 Tool 쪽에 show Kotlin bytecode 기능이 있네요 처음 봤습니다 ㅎㅎㅎ